.g8/datePage/app/viewmodels/checkAnswers/$className$Summary.scala

```diff
@@ -19,7 +19,7 @@ object $className$Summary  {
 
         SummaryListRowViewModel(
           key     = "$className;format="decap"$.checkYourAnswersLabel",
```

Comment:
this is the generator page - i would be tempted to keep this the same and not change it here unless we are certain that all summary list row views will want this

Reply:
This has changed as the variable was called dateTime, but it was only doing a date - it still has the same functionality and my new formatter now takes dateTime's place.

https://github.com/hmrc/trader-goods-profiles-frontend/blob/main/app/utils/DateTimeFormats.scala#L33

https://github.com/hmrc/trader-goods-profiles-frontend/pull/513/files#diff-4a77a0f36137ac7aa347aebe8835175592d6ccdd616d61e064639e81f78c6dd0R36